tools/packaging/kata-deploy/README.md

```diff
@@ -11,17 +11,28 @@ a node only if it uses either containerd or CRI-O CRI-shims.
 
 ### Install Kata on a running Kubernetes cluster
 
+#### Installing the latest image
+
+The latest image refers to pre-release and release candidate content.  For stable releases, please, use the "stable" instructions.
+
+```sh
+$ kubectl apply -f https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kata-deploy/kata-rbac/base/kata-rbac.yaml
```

Comment:
Does this really work?  I always had to use a GitHub raw link. If I curl that URL I don't get a YAML file but get a lot of HTML around a YAML file.  Same goes for all of the other https links below.

Reply:
You're right, I'll change all the links.